app/views/articles/_full_comment_area.html.erb

```diff
@@ -7,7 +7,7 @@
                 num: tag.span(t("views.articles.comments.num", num: @article.comments_count), class: "js-comments-count", data: { comments_count: @article.comments_count })) %>
         </h2>
         <div id="comment-subscription" class="print-hidden">
-          <div role="presentation" class="crayons-btn-group">
```

Comment:
When this HTML is rendered, there is only ever one button, so there is no need to add any additional semantics to place this in a group, and `role="presentation"` doesn't really have any impact on a `div` since it has no baked-in semantics